include/mxnet/imperative.h

```diff
@@ -35,48 +35,53 @@
 #include "./ndarray.h"
 
 namespace mxnet {
-/*! \brief runtime functions for NDArray */
-class Imperative {
+/*!
+ * Autograd Info used in class: nnvm::Node::info
+ */
+class AGInfo {
  public:
-  /*! \brief */
-  class AGInfo {
-   public:
-    Context ctx;
-    OpReqType grad_req;
-    OpStatePtr state;
-    std::vector<NDArray> outputs;
-    std::vector<NDArray> out_grads;
-    bool fresh_out_grad;
+  Context ctx;
```

Comment:
two spaces for indentation.

Reply:
Thanks, will fix.

Reply:
It's indendeted at two spaces already, public at one, as in other files, is this not correct?